governance/polls/Ratification Poll for Adding Sidestream Auction Services Core Unit, SAS-001 - November 8, 2021.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Ratification Poll for Adding Sidestream Auction Services Core Unit, SAS-001 - November 8, 2021
+summary: MIP39c2-SP25 adds SAS-001: Sidestream Auction Services Core Unit.
+discussion_link: https://forum.makerdao.com/t/mip39c2-sp25-adding-sidestream-auction-services-core-unit-sas-001/10796
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2021-11-08T16:00:00
+end_date: 2021-11-22T16:00:00
+---
+# Inclusion Poll for Adding Sidestream Auction Services Core Unit, SAS-001 - November 8, 2021
```

Comment:
_Ratification Poll for Adding Sidestream Auction Services Core Unit (SAS-001) - November 8, 2021_